### DISCUSSION TITLE
Solidity資源分享

### DISCUSSION THREAD

**Discussion by manipulateed** [1101-GS3073-C/Python-Community-GS3073-C#474] [General]
Solidity是一種合約導向式語言，被應用於各種不同的區塊鏈平台，以下的影片將帶大家進入這個領域，
我覺得這樣有系統的課程對於學習非常有幫助，希望大家能有所收穫。

https://www.youtube.com/watch?v=z2FEikbDqoA&t=6s

**Comment by williamyen0623** [comment]
想學習如何撰寫智能合約的基礎可以從Solidity開始

**Comment by boson13579** [comment]
可以發現solidity是針對區塊鏈裡知能合約的程式語言，可以先推個，不過，目前還沒打算參與區塊鏈的人，可以先不看。

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習
